tests/test_config.py

```diff
@@ -0,0 +1,52 @@
+import os
```

Comment:
Whoah, this is cool.

Reply:
Credit goes 100% to @kspeer825 - I copied this file almost 100% from tap-square where he wrote this test.